networking/eth2/src/testFixtures/java/tech/pegasys/teku/networking/eth2/Eth2NetworkFactory.java

```diff
@@ -92,7 +93,7 @@ public Eth2P2PNetworkBuilder builder() {
   }
 
   public void stopAll() {
-    networks.forEach(P2PNetwork::stop);
+    SafeFuture.allOf(networks.stream().map(P2PNetwork::stop).toArray(SafeFuture[]::new)).join();
```

Comment:
Waiter.waitFor is probably better than `.join()` here and below so the test doesn't hang forever if the future doesn't complete.

Reply:
Done.